### ISSUE TITLE
Nuget update apparently removed NodaTime references

### ISSUE THREAD

**Issue by mickeyjf** [nodatime/nodatime#1414]
I just updated my nuget NodaTime package to 2.4.7 and Npgsql.NodaTime to 4.0.10 from the immediately previous versions, and all references to NodaTime were removed from my .proj file. I was able to fix things by doing a diff with the old .csproj to discover that the references were in fact missing, and then manually updating the .proj. I am using Mono Develop 7.8.2, not VS. 

I found that solution here:
https://github.com/rianjs/ical.net/issues/402

Updates of other packages have not seemed to have this issue.

**Comment by jskeet** [comment]
When you say the "immediately previous versions" could you clarify *exactly* what those versions are? And which framework you're targeting?

This sounds like it's likely to be a MonoDevelop issue rather than something in the NuGet package. If you can provide a lot more detail I could potentially investigate, but installing MonoDevelop just for the sake of this issue feels a little over the top...

**Comment by jskeet** [comment]
I would suggest the first thing you should do is reproduce this in a tiny, tiny app that has no real purpose beyond demonstrating this.

Then see whether it still happens with just NodaTime instead of NodaTime and Npgsql.NodaTime.

**Comment by mickeyjf** [comment]
>  likely to be a MonoDevelop issue

Yes, if it were VS with so many more users you likely would have already heard about it if it were a real issue. I only posted here because the symptoms were the same as the problem I linked to that apparently had manifested on VS . 

The problem appeared when I had gone from NodaTime 2.4.6 to 2.4.7, updating using the Nuget Manager. A test program that contains no packages except NodaTime 2.4.6 built and ran, then also worked correctly when updated to 2.4.7. So it seems the NodaTime package is off the hook, anyway. It may be that the multiple updates I did at once confused the package manger  ?? 

For the record, Npgsql.NodaTime 4.0.9 was updated to 4.0.10. Interestingly, in my test program I got a "Type or namespace could not be found" with 4.0.9 after first installing it until I closed and opened the solution, but after updating to 4.0.10 there was no error message.

Thanks, sorry for the distraction.
